frontend/src/container/MetricsApplication/Tabs/External/External.tsx

```diff
@@ -251,6 +252,13 @@ function External({ getWidgetQueryBuilder }: ExternalProps): JSX.Element {
 					</Card>
 				</Col>
 			</Row>
+			<Row>
```

Comment:
don't render the entire row when service name is not available

Reply:
Fixed